frontend/test/pytest/test_decomposition.py

```diff
@@ -22,7 +22,7 @@
 # This is used just for internal testing
 from catalyst.pennylane_extensions import qfunc
 
-lightning = qml.device("lightning.qubit", wires=3)
+lightning = qml.device("lightning.kokkos", wires=3)
```

Comment:
It just uses kokkos list of operations because the one from lightning is a frozen set.